lib/bundler/settings.rb

```diff
@@ -138,7 +138,7 @@ def path
       if path = self[:path]
         "#{path}/#{Bundler.ruby_scope}"
       else
-        Bundler.rubygems.gem_dir
+        Pathname.new(File.dirname(__FILE__)).join(".bundle/gems")
```

Comment:
Except this is no longer using the RG path?

-Samuel E. Giddins

On Jun 19, 2015, at 12:46 PM, André Arko <notifications@github.com> wrote:

In lib/bundler/settings.rb:

> @@ -138,7 +138,7 @@ def path
>        if path = self[:path]
>          "#{path}/#{Bundler.ruby_scope}"
>        else
> -        Bundler.rubygems.gem_dir
> +        Pathname.new(File.dirname(__FILE__)).join(".bundle/gems")
@smlance the Ruby scope string defines the Ruby ABI (application binary interface), which breaks compatibility for compiled gems when it changes. The path gems are installed to always needs to include it.

@segiddins the Rubygems gem path already includes the scope string, so we don't need to add it ourselves.

—
Reply to this email directly or view it on GitHub.